src/main/java/de/doubleslash/keeptime/view/ManageProjectController.java

```diff
@@ -73,6 +74,14 @@ private void initialize() {
       sortIndexSpinner.getValueFactory().setValue(model.getAvailableProjects().size());
       formValidProperty.bind(Bindings.createBooleanBinding(() -> !nameTextField.getText().isBlank(),nameTextField.textProperty()));
       validateTextAlert.visibleProperty().bind(formValidProperty.not());
+      
+      Platform.runLater(() ->{
```

Comment:
let's also have a look on Tab-order of the controls. If cursor is in "project name" and I press tab, the cursor will be on "color" instead of the next field "description".
also in edit work in report dialog the tab-order is not right